doc/pages/user/045-device_errors.md

```diff
@@ -0,0 +1,76 @@
+# Device errors
+
+This page details the errors that can affect a device while it's sending data. The user can monitor
+device errors by installing a [device trigger](060-triggers.html#device-triggers) on `device_error`
+or checking the Devices tab in the Astarte Dashboard.
+
+## `write_on_server_owned_interface`
+
+The device is trying to write on a server owned interface.
+
+## `invalid_interface`
+
+The interface name received in the message is invalid.
+
+## `invalid_path`
+
+The path received in the message is invalid.
+
+## `mapping_not_found`
+
+The path received in the message can't be found in the interface mappings. This could be the result
+of an outdated interface.
```

Comment:
This could be the result of an outdated interface --> better description would explain:
1. the device has a more recent interface than what is currently installed on Astarte
2. the interface has same name and version but they are different.